app/src/main/java/org/mifos/mobile/ui/fragments/AboutUsFragment.kt

```diff
@@ -1,90 +1,41 @@
 package org.mifos.mobile.ui.fragments
 
-import android.content.Intent
-import android.net.Uri
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import androidx.compose.ui.platform.ComposeView
 import dagger.hilt.android.AndroidEntryPoint
-import org.mifos.mobile.BuildConfig
-import org.mifos.mobile.R
-import org.mifos.mobile.databinding.FragmentAboutUsBinding
-import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.screens.AboutView
 import org.mifos.mobile.ui.fragments.base.BaseFragment
-import java.util.Calendar
 
 /*
 ~This project is licensed under the open source MPL V2.
 ~See https://github.com/openMF/self-service-app/blob/master/LICENSE.md
 */
 @AndroidEntryPoint
 class AboutUsFragment : BaseFragment() {
-    private var _binding: FragmentAboutUsBinding? = null
-    private val binding get() = _binding!!
 
-    private val licenseLink = "https://github.com/openMF/mifos-mobile/blob/development/LICENSE.md"
-    private val sourceCodeLink = "https://github.com/openMF/mifos-mobile"
-    private val websiteLink = "https://openmf.github.io/mobileapps.github.io/"
+    private lateinit var composeView: ComposeView
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
-        _binding = FragmentAboutUsBinding.inflate(inflater, container, false)
-        val rootView = binding.root
-        setToolbarTitle(getString(R.string.about_us))
-        binding.appVersion.text = BuildConfig.VERSION_NAME
-        binding.tvCopyRight.text =
-            getString(R.string.copy_right_mifos, Calendar.getInstance()[Calendar.YEAR].toString())
-        return rootView
+        return ComposeView(requireContext()).also {
+            composeView = it
+        }
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        binding.aboutWebsiteContainer.setOnClickListener {
-            showWebsite()
-        }
-
-        binding.aboutLicensesContainer.setOnClickListener {
-            showOpenSourceLicenses()
-        }
-
-        binding.aboutPrivacyPolicyContainer.setOnClickListener {
-            showPrivacyPolicy()
+        composeView.setContent {
```

Comment:
As we are using Compose with an existing view system, you should consider using `ViewCompositionStrategy`, for more details, take a look at [this document](https://developer.android.com/jetpack/compose/migrate/interoperability-apis/compose-in-views), please.